soperator/modules/slurm/templates/helm_values/terraform_fluxcd_values.yaml.tftpl

```diff
@@ -0,0 +1,713 @@
+                customInitContainers:
+                  - name: ensure-jail-virtiofs
+                    image: cr.eu-north1.nebius.cloud/soperator/busybox
+                    volumeMounts: [{ name: jail, mountPath: /volume-mount }]
+                    command: ["grep", " /volume-mount virtiofs ", "/proc/mounts"]
+
+              worker:
+                size: ${slurm_cluster.nodes.worker.size}
+                k8sNodeFilterName: ${slurm_cluster.k8s_node_filters.worker.name}
+                cgroupVersion: v2
+                enableGDRCopy: true
+                slurmNodeExtra: "${slurm_cluster.nodes.worker.slurm_node_extra}"
+                sshdConfigMapRefName: "${slurm_cluster.nodes.worker.sshd_config_map_ref_name}"
+                supervisordConfigMapRefName: custom-supervisord-config
+                slurmd:
+                  resources:
+                    cpu: ${slurm_cluster.nodes.worker.resources.cpu * 1000}m
+                    memory: ${slurm_cluster.nodes.worker.resources.memory}Gi
+                    ephemeralStorage: ${slurm_cluster.nodes.worker.resources.ephemeral_storage}Gi
+                    gpu: ${slurm_cluster.nodes.worker.resources.gpus}
+                munge:
+                  resources:
+                    cpu: ${slurm_cluster.nodes.munge.resources.cpu * 1000}m
+                    memory: ${slurm_cluster.nodes.munge.resources.memory}Gi
+                    ephemeralStorage: ${slurm_cluster.nodes.munge.resources.ephemeral_storage}Gi
+                customInitContainers:
+                  - name: ensure-jail-virtiofs
+                    image: cr.eu-north1.nebius.cloud/soperator/busybox
+                    volumeMounts: [{ name: jail, mountPath: /volume-mount }]
+                    command: ["grep", " /volume-mount virtiofs ", "/proc/mounts"]
+                  %{~ for sub_mount in slurm_cluster.jail_submounts ~}
+                  - name: ensure-jail-submount-${sub_mount.name}-virtiofs
+                    image: cr.eu-north1.nebius.cloud/soperator/busybox
+                    volumeMounts: [{ name: '${sub_mount.name}', mountPath: /volume-mount }]
+                    command: ["grep", " /volume-mount virtiofs ", "/proc/mounts"]
+                  %{~ endfor ~}
+                volumes:
+                  spool:
+                    volumeClaimTemplateSpec: null
+                    volumeSourceName: worker-spool
+                  jailSubMounts:
+                    %{~ if slurm_cluster.nfs.enabled ~}
+                    - mountPath: ${slurm_cluster.nfs.mount_path}
+                      name: nfs
+                      volumeSourceName: nfs
+                    %{~ endif ~}
+                    %{~ for sub_mount in slurm_cluster.jail_submounts ~}
+                    - name: ${sub_mount.name}
+                      mountPath: ${sub_mount.mount_path}
+                      volumeSourceName: jail-submount-${sub_mount.name}
+                    %{~ endfor ~}
+                  customMounts:
```

Comment:
Node-local related mounts are lost:
https://github.com/nebius/nebius-solution-library/blob/release/soperator/soperator/modules/slurm/templates/helm_values/slurm_cluster.yaml.tftpl#L385-L396

Reply:
This was meant to be just for line 474